### ISSUE TITLE
Double Concentration on Grandfathered Games

### ISSUE THREAD

**Issue by justinthejaguar** [death-save/combat-utility-belt#331]
**Describe the bug**
While using the Concentrator, the condition appears to be a different form of concentration offered in the conditions lab. This form is not able to be removed unless the user has access to Active Effects. Attempts below were completed with only CUB active as well as with DAE on another attempt for photos.

**To Reproduce**
1. Open any prior game where the Concentrator and Conditions Lab were active prior to the 0.7.5 release
2. Make sure that the Concentrator and Conditions Lab are active for current release.
3. Cast a concentration spell such as Hex.
4. Attempt to remove the Concentration Active Effect from the token.

**Expected behavior**
The condition icon selector attached to the token will not show that the Concentration effect is active. Attempting to disable will produce a second concentration effect on the token.

**Screenshots**
<img width="717" alt="Screen Shot 2020-10-29 at 7 06 49 PM" src="https://user-images.githubusercontent.com/38022481/97643397-ea2b1d80-1a1d-11eb-9af9-fb475623eb71.png">
<img width="123" alt="Screen Shot 2020-10-29 at 7 25 10 PM" src="https://user-images.githubusercontent.com/38022481/97643402-ebf4e100-1a1d-11eb-85c8-692d70a71d82.png">
<img width="291" alt="Screen Shot 2020-10-29 at 7 33 02 PM" src="https://user-images.githubusercontent.com/38022481/97643403-ec8d7780-1a1d-11eb-8faf-ebc5ae92a3a7.png">

**Environment Info (please complete the following information):**
 - Foundry Version: 0.7.5
 - CUB Version 1.3.3
 - Browser Chrome and Host
- OS Mac Os

**Additional context**
It is of note that the module works in a test environment with no issue. New worlds are fine.

**Comment by eclarke12** [comment]
Please read the notes here and let me know if it resolves the issue:
https://github.com/death-save/combat-utility-belt/blob/master/CHANGELOG.md#known-issues

**Comment by Sniperfox47** [comment]
If the note you're referring to is the one instructing to save the condition assignment that hasn't resolved it for me. And in my case it's reproducable by importing a saved condition lab mapping such as the attached. 
[cub-dnd5e-condition-map.zip](https://github.com/death-save/combat-utility-belt/files/5468898/cub-dnd5e-condition-map.zip)

**Comment by eclarke12** [comment]
Due to the high likelihood of causing larger issues, no fix will be provided for this, however there is a 2 part workaround that seems to be very successful:
1. Go into the Condition Lab and click Save to ensure it saves the new Active Effect info
2. Delete any tokens with the old "non-Active" Concentration and drop a new version on the scene

**Comment by aarongarrett** [comment]
I think this is the problem I'm facing. I have player tokens that have the concentration marker on them, and there seems to be no way to remove it. If I click that marker in the marker list, it just adds a new one under the old one (though the new one does allow toggling to remove). I've followed those 2 instructions, but even the newly re-added tokens have the concentration symbol as before. I have removed every token from every scene, gone into the Condition Lab, clicked Save Changes, and added a new version of the token. It still has the concentration symbol. What is the fix for this?
